lib/logger.js

```diff
@@ -4,8 +4,12 @@ const options = require('./options')
 const logger = new winston.Logger({
   level: options.logLevel,
   transports: [
-    new (winston.transports.Console)({timestamp: true, colorize: true})
-  ]
+    new (winston.transports.Console)({ timestamp: true, colorize: true })
+  ],
+  rewriters: [(_, __, meta) => {
```

Comment:
Here's a way to not require dummy variables:

```suggestion
  rewriters: [(...[,,meta]) => {
```